pyblazing/pyblazing/apiv2/context.py

```diff
@@ -1416,6 +1424,9 @@ def __init__(
                     It should use what the user set. If the user does not explicitly set it,
                     by default it will be set by whatever dask client is using ('tcp', 'ucx', ..).
                     NOTE: This parameter only works when used in the BlazingContext.
+            CURRENT_TIMESTAMP: A string representing the current timestamp in order to get the same
```

Comment:
I dont think the CURRENT_TIMESTAMP should be a `config_option`. Its not something we want to have a user be able to set. Its also not really optional.

I think it should be something that is passed in like ctxToken, and similarly, like it, be added to the `Context`